src/plugins/opentypejs.plugin.cff1file.js

```diff
@@ -0,0 +1,75 @@
+/*!
+ * OpenType.js plugin to support parsing of standalone CFF1 font files
+ * 
+ * https://adobe-type-tools.github.io/font-tech-notes/pdfs/5176.CFF.pdf
+ * 
+ * opentype.js
+ * https://github.com/opentypejs/opentype.js
+ * (c) 2015-present Frederik De Bleser and contributors
+ * opentype.js may be freely distributed under the MIT license.
+ */
+
+let isResponsible = new WeakMap();
+
+const plugin_cff1file = {
+    parseBuffer_signature: function(returnData, params) {
```

Comment:
I think there are two ways of declaring a method:
```js
parseBuffer_signature: function(returnData, params) {
```

```js
parseBuffer_signature(returnData, params) {
```

one of them bind the `this` at creation , the other at runtime
(maybe this would help avoid using those .call() usage ?)

Reply:
Good point, I'll take a look at it